__collection-name__: model-parameters
body_com  :  [0.0273, -0.000242, 0.0143]
body_mass  :  9.9
body_inertia  :  [0.0158533, -0.000173, -0.0102, -0.000173, 0.0377999, -2.83e-05, -0.0102, -2.83e-05, 0.0456542]

body_length  :  0.164
body_width  :  0.042
abad_torque_max  :  48
abad_lower_bound  :  -0.6806784082777885
abad_upper_bound  :  0.6806784082777885
abad_com  :  [-0.00392, -0.015, -0.000306]
abad_mass  :  1.054
abad_inertia  :  [0.000469246, 2.7e-05, -3.44e-07, 2.7e-05, 0.000807490, -1.95e-06, -3.44e-07, -1.95e-06, 0.000552929]
abad_rotor_location  :  0.0642303
rotor_mass  :  0.0567
rotor_inertia_z  :  [2.53e-05, 0, 0, 0, 2.53e-05, 0, 0, 0, 4.78e-05]
abad_link_length  :  0.114    # hip_offset

hip_torque_max  :  48
front_hip_lower_bound  :  -2.792526803190927
front_hip_upper_bound  :  1.3264502315156903
hip_com  :  [0.00212, -0.0212, -0.0184]
hip_mass  :  1.43
hip_inertia  :  [0.005529065, -2.23e-05, 3.85e-05, -2.23e-05, 0.005139339, 0.000199, 3.85e-05, 0.000199, 0.001367788]
hip_rotor_location  :  0.07577
knee_rotor_location  :  0.0342303
hip_link_length  :  0.2    # hip_length

knee_torque_max  :  48
knee_lower_bound  :  0.5235987755982988
knee_upper_bound  :  2.5307274153917776
knee_com  :  [-0.0006, 4.72e-05, -0.0893]
knee_mass  :  0.781
knee_inertia  :  [0.002997972, 3e-09, -2.37e-05, 3e-09, 0.003014022, -6.03e-07, -2.37e-05, -6.03e-07, 0.000032426]
knee_link_length  : 0.208     # knee_length - foot_radius / 2

foot_radius  :  0.0225

abad_gear_ratio : 9.
hip_gear_ratio : 9.
knee_gear_ratio : 9.

hip_cover_location : 0.025
knee_rubber : 0.02
rear_hip_lower_bound  :  -3.141592653589793
rear_hip_upper_bound  :  0.9773843811168246
nose_location : [0.294, 0.03, 0.120]
ear_location : [0.225, 0.04, 0.2]